backend/docker-compose.yml

```diff
@@ -2,7 +2,11 @@ version: '3'
 

```

Comment:
This specifies to the docker-compose config file that we won't use the default dockerfile but a specific one that we define bellow. Context means the folder in which the code and dockerfile is located. Dockerfile is the path of the dockerfile regarding the context.